core/src/main/java/com/aem/demo/core/models/MainPageModel.java

```diff
@@ -100,4 +106,27 @@ private List<NewsCardModel> separateModels(List<NewsCardModel> list, int paginat
         return separatedPart;
     }
 
+    private void paginateModels(List<NewsCardModel> models) {
+        double listSize = models.size();
+        double listPart = listSize / 5;
+        double roundPart = Math.ceil(listPart);
+
+        pageNumbers = IntStream.range(1, (int) (roundPart + 1)).boxed().collect(Collectors.toList());
```

Comment:
so what does this method do?
you don't return any value here

Reply:
it calculates the sequence of numbers that should be created per pages(5).
I will change the name of it)